### ISSUE TITLE
reset detailed statistics

### ISSUE THREAD

**Issue by broomdodger** [tgraf/bmon#33]
Is it possible to reset the detailed statistics?
Specifically bytes RX, reset to 0.

**Comment by tgraf** [comment]
The statistics as presented are read directly from the kernel. The only way I'm aware of the reset the statistics is to unload and then reload the driver kernel module of the particular NIC.

**Comment by tgraf** [comment]
I'm happy to take a patch to the curses or other modes which stores the current counter state and emulates a reset in user space. That reset would only be effective while bmon is running.

**Comment by broomdodger** [comment]
I was thinking of a variable initially set to zero.
Is this what your recent comment suggests?

On a reset request the current value of RX/TX bytes
is stored in a variable effectively starting at zero.

Something like:
`RX_current - $RX_reset`

On a restart of bmon `set RX_reset=0`

Unfortunately I do not know enough to do the patch.

**Comment by tgraf** [comment]
Yes, that's what I meant as well. I realized this is actually very simple to implement so I did.
Please try out commit 4efaa8ce9e82777d1208688e5bad833043587d81 and let me know if this works for you.

**Comment by broomdodger** [comment]
Lubuntu 15.10 32-bit
`bmon-4efaa8ce9e82777d1208688e5bad833043587d81.zip`
```
sudo apt-get install build-essential make libconfuse-dev libnl-3-dev libnl-route-3-dev libncurses-dev
Reading package lists... Done
Building dependency tree
Reading state information... Done
Note, selecting 'libncurses5-dev' instead of 'libncurses-dev'
build-essential is already the newest version.
libncurses5-dev is already the newest version.
libnl-3-dev is already the newest version.
libnl-route-3-dev is already the newest version.
make is already the newest version.
libconfuse-dev is already the newest version.
0 upgraded, 0 newly installed, 0 to remove and 2 not upgraded.
```
Then:
```
./autogen.sh
./autogen.sh: line 3: autoreconf: command not found
```
ok something else is needed:
```
sudo apt-get install autoconf
```
oops missing:
```
./configure: line 6987: syntax error near unexpected token `CONFUSE,'
./configure: line 6987: `PKG_CHECK_MODULES(CONFUSE, libconfuse, , as_fn_error $? "requires libconfuse" "$LINENO" 5)'
```
but these are installed:
```
libconfuse-common
libconfuse-dev
libconfuse0
```
What am I missing?

**Comment by tgraf** [comment]
It looks like you don't have pkg-config installed.

**Comment by tgraf** [comment]
I've added a check to configure to catch this.

**Comment by broomdodger** [comment]
ok!
It compiles, quick test it runs and `r` seems to work.
But now I must run to work.
More later.

**Comment by broomdodger** [comment]
Thank you... I like the new `'r'` function.
Lubuntu 15.10
This is what I needed to compile bmon adding `autoconf pkg-config`
```
sudo apt-get install build-essential make libconfuse-dev libnl-3-dev libnl-route-3-dev libncurses-dev autoconf pkg-config
```

**Comment by tgraf** [comment]
Resolved in commit 4efaa8ce9e82777d1208688e5bad833043587d81
